scan/pki.go

```diff
@@ -94,6 +95,7 @@ func chainValidation(host string) (grade Grade, output Output, err error) {
 		return
 	}
 
+	// This is probably a redundant check
```

Comment:
Style nitpicks: this should be a complete sentence. It would also be useful to explain why it might be redundant and why it's been kept in (even if the answer is just that the comment serves as a marker to go back and look at the code later).